docs/internals/contributing/writing-documentation.txt

```diff
@@ -45,11 +45,16 @@ documentation is transformed into HTML, PDF, and any other output format.
 __ https://www.sphinx-doc.org/
 __ https://docutils.sourceforge.io/
 
-To build the documentation locally, install Sphinx:
+To build the documentation locally, from the project root, install the
+documentation building requirements:
 
 .. console::
 
-     $ python -m pip install Sphinx
+     $ python -m pip install -r docs/requirements.txt
+
+This command must be run (from the project root) because the docs are dependent
```

Comment:
The wording looks code but I meant to move the parentheses around this paragraph, not around "from the project root." The parentheses help clarify that this is an explanatory paragraph of why the command needs to be run "from the project root" as already mentioned in the preceding paragraph.